app/assets/javascripts/modules/form-card-type.js

```diff
@@ -135,12 +135,12 @@ var showCardType = function() {
     };
   }
 
-  var checkCardtypeIsAllowed = function () {
+      var checkCardtypeIsAllowed = function () {
     return new Promise(function (resolve, reject) {
       var request = new XMLHttpRequest();
       var card = getCardType();
       // this should already be picked up by the other validations
```

Comment:
This comment is out of date now because nowhere prior to this does the length 11 check